tests/unit_tests/data_steward/utils/bq_test.py

```diff
@@ -262,3 +262,24 @@ def test_copy_datasets(self, mock_client, mock_list_tables,
                          f'{self.dataset_id}_snapshot')
         mock_list_tables.assert_called_once_with(self.dataset_id)
         self.assertEqual(mock_copy_table.call_count, len(list_tables_results))
+
```

Comment:
If the PR is refactored (and the refactoring is agreeable to everyone), these tests may not be necessary.